CHANGELOG.md

```diff
@@ -6,6 +6,17 @@
 
 - Added support for `delimiters` parameter in `functions.initcap()`.
 - Added support for `functions.hash()` to accept a variable number of input expressions.
+- Added support for `session.getOrCreate` and `session.appName`
+- Added support for `functions.array_distinct`
+- Added support for alias `Column.getField`
+- Added support for `functions.regexp_extract`
+- Added support for `functions.struct`
+- Added support for `functions.daydiff` which is compatible with spark datediff
+- Added support for `functions.date_add` and `functions.date_sub` to ease common day add/substract operations
+- Added support for `functions.format_number`
+- Added support for `functions.bround`
+- Added API `Session.RuntimeConfig` for getting/setting/checking the mutability of any runtime configuration.
+- Added support managing case sensitivity in `Row` results from `DataFrame.collect` using `case_sensitive` parameter.
```

Comment:
since the 1.3.0 release has gone out. These would be moved to a new section with release 1.4.0